lib/elixir/src/elixir_map.erl

```diff
@@ -217,8 +217,10 @@ format_error({non_map_after_struct, Expr}) ->
   io_lib:format("expected struct to be followed by a map, got: ~ts",
                 ['Elixir.Macro':to_string(Expr)]);
 format_error({invalid_struct_return_value, Module, Arity, Expr}) ->
-  io_lib:format("expected ~ts.__struct__/~p to return a map, got: ~ts",
-                ['Elixir.Macro':to_string(Module), Arity, 'Elixir.Macro':to_string(Expr)]);
+  Message =
+    "expected ~ts.__struct__/~p to return a map with a :__struct__ key that holds the "
+    "name of the struct (atom), got: ~ts",
+  io_lib:format(Message, ['Elixir.Macro':to_string(Module), Arity, 'Elixir.Macro':to_string(Expr)]);
```

Comment:
```suggestion
  io_lib:format(Message, [elixir_aliases:inspect(Module), Arity, 'Elixir.Macro':to_string(Expr)]);
```

Reply:
I will change other occurrences in a different PR.

Reply:
Although I can see we used `Elixir.Macro:to_string` before .